Source/WebCore/rendering/RenderSearchField.cpp

```diff
@@ -191,8 +195,10 @@ void RenderSearchField::updateFromElement()
     if (cancelButtonElement())
         updateCancelButtonVisibility();
 
-    if (m_searchPopupIsVisible)
-        m_searchPopup->popupMenu()->updateFromElement();
+    if (m_searchPopupIsVisible) {
+        RefPtr popupMenu = m_searchPopup->popupMenu();
+        popupMenu->updateFromElement();
```

Comment:
`protectedSearchPopup()->protectedPopupMenu()->updateFromElement();`. We shouldn't need a new local variable.